test/markdown_test.rb

```diff
@@ -68,10 +68,13 @@ def setup
   end
 end
 
-# Will generate RDiscountTest, KramdownTest, etc.
-markdown_templates = Tilt.lazy_map['md'].map { |klass, _require_path| klass }
-
-markdown_templates.each do |template_name|
+[
```

Comment:
Instead of loading all possible markdown engines from Tilt, we require those that we support